### PR TITLE
Implement Info Pane

### PR DESCRIPTION
**Pull request by giraugh**
image

### CODE REVIEW

**Review by tdib — APPROVED** [review]
LGTM

File: frontend/src/components/Sidepanel/Panels/Info/Info.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      for (let symbol of alphabet) {
+        const transitions = validTransitions(resolvedGraph, state.id, symbol)
+        for (let { transition } of transitions) {
+          map[[state.id, symbol]] = Array.from(new Set([...map[[state.id, symbol]] ?? [], transition.to]))
+        }
+      }
+    }
+
+    return map
+  })
```
Comment (GRA0007) [inline frontend/src/components/Sidepanel/Panels/Info/Info.js:43]: All of these `useMemo`s are missing deps, making them ineffective

File: frontend/src/components/Sidepanel/Panels/Info/infoStyle.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    &:nth-last-child(1) {
+      border-right: 1px solid var(--table-border-color);
+    }
+  }
+
+   th {
+    background: var(--toolbar);
+    font-weight: 400;
+    border-color: white;
```
Comment (GRA0007) [inline frontend/src/components/Sidepanel/Panels/Info/infoStyle.js:71]: This needs to be `var(--table-border-color)`

**Review by GRA0007 — CHANGES_REQUESTED** [review]
Looks good!

**Review by GRA0007 — APPROVED** [review]
Cheese gromit

**Pull request merged**